flexy-osisoft-connector/src/com/hms/flexyosisoftconnector/Main.java

```diff
@@ -81,5 +82,20 @@ public static void setCertificatePath(String path) {
          System.out.println("Error: Setting certificate directory failed");
          System.exit(0);
       }
+   }
+
+   // Sets the timeouts

```

Comment:
are these timeouts specific to this particular jvm instance or system wide?

Reply:
System wide unfortunately.

Reply:
Should note that in the comment for that method so the caller knows that you are changing that setting. It may greatly impact script behavior to have such a short timeout. Might also make sense to add a note about how that should be a feature request to support those timeouts on a per connection basis and pass that along to Tom.

Reply:
Added comment